src/main/java/io/stargate/sgv2/jsonapi/api/model/command/builders/TableSortClauseBuilder.java

```diff
@@ -14,4 +21,50 @@ public TableSortClauseBuilder(TableSchemaObject table) {
   public SortClause buildAndValidate(ObjectNode sortNode) {
     return defaultBuildAndValidate(sortNode);
   }
+
+  @Override
+  protected String validateSortClausePath(String path) {
+    // Tables have few rules: but cannot be empty
+    if (path.isEmpty()) {
```

Comment:
the collection builder runs naming rules, why dont we here ? e.g. the TableNamingRule is pretty basic. and reading that class and here, should it be isEmpty or isBlank ?

Reply:
`isEmpty()` works here since caller does `trim()`. Naming rules avoided (but not commented) just to allow leading `$`. But should we? If I remember correctly, they would cause issues elsewhere.